### PR TITLE
Feature/154 external housing available read endpoint

### PR DESCRIPTION
**Pull request by JaniceHe26**
Added read one route and controller endpoint for external housing available.

### CODE REVIEW

**Comment by JaniceHe26** [comment]
Screen Shot 2020-08-17 at 4 25 58 PM

File: lib/external-housing-available/external-housing-available.controllers.js [hunk 1]
```diff
@@ -202,6 +210,16 @@ module.exports = (models) => {
...
   }
 
+  function getOneExternalHousingAvailable(id) {
+    return models.ExternalHousingAvailable.findByPk(id)
+    .then(response => {
+      return response;
+    })
+    .catch(error => {
+      return error;
+    });
```
Comment (zaclittleberry) [inline lib/external-housing-available/external-housing-available.controllers.js]: remember, we don't want to use then or catch in our db helper functions anymore. catching here would actually prevent us from entering the catch in our try...catch where we'd properly throw a boom error. This is something that needs to be cleaned up elsewhere at another time, but for now we at least don't want to grow our problem.

**Review by zaclittleberry — CHANGES_REQUESTED** [review]
just that small change with the .then .catch and then it looks all good.

**Review by zaclittleberry — APPROVED** [review]

**Review by zaclittleberry — CHANGES_REQUESTED** [review]
you made an un-needed addition! :P I am in favor of semicolons practically everywhere they are optional, but in this case no semi-colon goes there. It shouldn't break anything, mainly because the parser should ignore it, but since these functions aren't being assigned to a variable, there is no need for semicolon. This stackoverflow answers it better: https://stackoverflow.com/a/1834674

**Review by zaclittleberry — APPROVED** [review]

**Pull request merged**